146102.out

```diff
@@ -0,0 +1,401 @@
+Extending Research Protocols for Organization(146102) Australian Prostate Centre
```

Comment:
Should these logs be attached to the PR #3986 instead of committed?...

Reply:
that's just a test log, for you to verify the data isn't being modified beyond format.  i don't see value in keeping it with the story.

Reply:
Ah!  those files weren't to be included at all.  Thanks, will delete.